kube-runtime/src/controller/mod.rs

```diff
@@ -309,10 +309,12 @@ where
 
     /// Indicate child objets `K` owns and be notified when they change
     ///
-    /// This type `Child` must have `OwnerReference`s set to point back to `K`.
+    /// This type `Child` must have [`OwnerReference`] set to point back to `K`.
     /// You can customize the parameters used by the underlying `watcher` if
     /// only a subset of `Child` entries are required.
     /// The `api` must have the correct scope (cluster/all namespaces, or namespaced)
+    ///
+    /// [`OwnerReference`]: https://docs.rs/k8s-openapi/0.10.0/k8s_openapi/apimachinery/pkg/apis/meta/v1/struct.OwnerReference.html
```

Comment:
Couldn't use the intra-docs syntax because `kube-runtime` doesn't depend on `k8s-openapi`.